pom.xml

```diff
@@ -345,29 +449,37 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
-        <version>2.17</version>
+        <version>3.0.0</version>
         <executions>
           <execution>
-            <id>verify-style</id>
-            <phase>process-classes</phase>
             <goals>
               <goal>check</goal>
             </goals>
           </execution>
         </executions>
         <configuration>
-          <failsOnError>false</failsOnError>
-          <failOnViolation>false</failOnViolation>
-          <logViolationsToConsole>true</logViolationsToConsole>
-          <configLocation>https://raw.githubusercontent.com/folio-org/raml-module-builder/master/codestyles.xml</configLocation>
-          <cacheFile>${basedir}/target/cachefile</cacheFile>
+          <configLocation>google_checks.xml</configLocation>
```

Comment:
Should propose adopting the Google code standard rather than the Oracle/Sun one?

It would make areas like this easier.

Reply:
Probably. I think the biggest difference from what we are doing now, is that google allows line width up to 100 chars and import ordering is alphabetical.

Reply:
Does it care about indent size? FOLIO chose 2 spaces instead of 4 in the early days (mostly due to nested callbacks)

Reply:
Yes, it does, but Google uses 2 spaces as the indent size, so we are already in line there.

Reply:
Raised an [issue](https://issues.folio.org/browse/FOLIO-2127) proposing this